main.js

```diff
@@ -1,4 +1,5 @@
 const bookForm = document.querySelector('#form');
+const collection = document.querySelector('.table');
 
 function Book() {
   this.title = document.querySelector('#title').value;
```

Comment:
- [ ] Following the project requirements, you should use classes instead of objects and arrays.
```js
class Book() {
  constructor() {
  }
}
```

- [ ] You should also create class methods to add and remove books.